### ISSUE TITLE
Syntax error or access violation: 1101 BLOB, TEXT, GEOMETRY or JSON column 'uptime_check_failure_reason' can't have a default value

### ISSUE THREAD

**Issue by vesper8** [spatie/laravel-uptime-monitor#169]
Getting this error on Mysql 5.7.25

```
Syntax error or access violation: 1101 BLOB, TEXT, GEOMETRY or JSON column 'uptime_check_failure_reason' can't have a default value
```

Caused by 
```
$table->text('uptime_check_failure_reason')->default('');
```

I see this field was changed from string to text in a PR a few months ago. Surprised no one's reported this yet

Changing to
```
$table->text('uptime_check_failure_reason')->nullable();
```

Fixes the issue. 

And that's ok because this field is initialized to '' when Monitor::create() is called

**Comment by sixlive** [comment]
I'll take a look at this today or tomorrow.

**Comment by sixlive** [comment]
I was able to confirm this is an issue. `$table->text('uptime_check_failure_reason')->nullable();` seems like an appropriate fix. If you'd like credit for the PR you can open one or I can make one later today.

**Comment by vesper8** [comment]
you can go ahead : ) thanks for the quick fix

**Comment by sixlive** [comment]
Released with [3.4.1](https://github.com/spatie/laravel-uptime-monitor/releases/tag/3.4.1)
